csfieldguide/templates/interactives/distance-from-stereo-images.html

```diff
@@ -0,0 +1,115 @@
+    </div>
+    <div class="col-sm-6">
+      <h4 class="text-center mb-2">{% trans "Right Camera" %}</h4>
+      <canvas id="canvas-right"></canvas>
+      <img id="img-right" src="{% static 'interactives/distance-from-stereo-images/images/lighter-right.jpg' %}">
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col-sm-6">
+      <p>{% trans "Upload a custom left image:" %}</p>
+      <div class="input-group mb-3">
+        <div class="custom-file">
+          <input type="file" class="custom-file-input" id="stereo-left-input" accept="image/*">
+          <label class="custom-file-label" for="stereo-left-input">{% trans "Choose file" %}</label>
+        </div>
+      </div>
+    </div>
+    <div class="col-sm-6">
+      <p>{% trans "Upload a custom right image:" %}</p>
+      <div class="input-group mb-3">
+        <div class="custom-file">
+          <input type="file" class="custom-file-input" id="stereo-right-input" accept="image/*">
+          <label class="custom-file-label" for="stereo-right-input">{% trans "Choose file" %}</label>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col-sm-6">
+      <p>Left Width (x<sub>0</sub>): <span id="left-width"></span>px</p>
+      <p>Left X (x<sub>1</sub>): <span id="left-x"></span></p>
+      <p>Left Y: <span id="left-y"></span></p>
+    </div>
+    <div class="col-sm-6">
+      <p>Right Width (x<sub>0</sub>): <span id="right-width"></span>px</p>
+      <p>Right X (x<sub>2</sub>): <span id="right-x"></span></p>
+      <p>Right Y: <span id="right-y"></span></p>
+    </div>
+  </div>
+
+  <div class="input-group mb-3">
+    <div class="input-group-prepend">
+      <span class="input-group-text" id="angle-label">Angle of view (&theta;<sub>0</sub>)</span>
+    </div>
+    <input id="angle-of-view" type="number" step="0.01" class="form-control" value="63.18" autocomplete=off aria-label="half-angle" aria-describedby="angle-label">
+    <div class="input-group-append">
+      <span class="input-group-text" id="angle-unit">Degrees</span>
+    </div>
+  </div>
+
+  <div class="input-group mb-3">
+    <div class="input-group-prepend">
+      <span class="input-group-text" id="distance-label">Distance between pictures (B)</span>
+    </div>
+    <input id="camera-distance" type="number" step="0.01" class="form-control" value="0.1" autocomplete=off aria-label="camera-distance" aria-describedby="distance-label">
+    <div class="input-group-append">
+      <button class="btn btn-outline-secondary dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" id="distance-unit">Metres</button>
+      <div class="dropdown-menu" id="distance-unit-list">
+        <a class="dropdown-item" href="#">Metres</a>
+        <a class="dropdown-item" href="#">Inches</a>
+      </div>
+    </div>
+  </div>
+
+  <div id="result-div" class="container">
+    <button id="go-button" class="btn btn-outline-primary">Go!</button>
```

Comment:
Would recommend changing the label to inform the user what is being done. You will want to change any references in the text, plus you may want to change ID of element as well.

```suggestion
    <button id="go-button" class="btn btn-outline-primary">Calculate distance</button>
```

Reply:
Renamed wording and id.